zsh throws error when using `--update`

```source ~/.local/bin/license --update
Kong Enterprise Edition licensed to Kong Inc., license expires: 2019-01-01
[WARNING] The license has expired!
Logging into 1Password...
Enter the password for aaron@konghq.com at team-kong.1Password.com:
parse error: Invalid string: control characters from U+0000 through U+001F must be escaped at line 3, column 97
parse error: Invalid string: control characters from U+0000 through U+001F must be escaped at line 3, column 97
parse error: Invalid string: control characters from U+0000 through U+001F must be escaped at line 3, column 97
Downloading license...
[ERROR] failed to download the Kong Enterprise license file
    Repository path missing or not specified.

switching to /bin/bash to run the update is the workaround for this.

--- a/license.sh
+++ b/license.sh
@@ -207,9 +207,9 @@ fi
 op signout --session="$OP_TOKEN"
 
 
-BINTRAY_APIKEY=$(echo "$DETAILS" | jq '.details.fields[]? | select(.designation=="password").value' | sed s/\"//g)
-BINTRAY_USERNAME=$(echo "$DETAILS" | jq '.details.fields[]? | select(.designation=="username").value' | sed s/\"//g)
-BINTRAY_REPO=$(echo "$DETAILS" | jq '.details.sections[].fields[]? | select(.t=="Package/Repository").v' | sed s/\"//g)
+BINTRAY_APIKEY=$(printf "%s" "$DETAILS" | jq '.details.fields[]? | select(.designation=="password").value' | sed s/\"//g)
+BINTRAY_USERNAME=$(printf "%s" "$DETAILS" | jq '.details.fields[]? | select(.designation=="username").value' | sed s/\"//g)
+BINTRAY_REPO=$(printf "%s" "$DETAILS" | jq '.details.sections[].fields[]? | select(.t=="Package/Repository").v' | sed s/\"//g)
 
 
 echo "Downloading license..."

